client/src/main/java/io/pravega/keycloak/client/KeycloakAuthzClient.java

```diff
@@ -128,7 +128,7 @@ public String getRPT() {
         return e -> {
             Throwable rootCause = unwrap(e);
             if (rootCause instanceof HttpResponseException) {
-                int statusCode = ((HttpResponseException) e).getStatusCode();
+                int statusCode = ((HttpResponseException) rootCause).getStatusCode();
```

Comment:
This was a real bug found by @crazyzhou.   We were checking the rootCause, but casting the original exception.  This only worked when the original exception and its rootCause were one and the same.